e2e/settings.ini

```diff
@@ -0,0 +1,7 @@
+[DEFAULT]
```

Comment:
Can you help to do one more experiment:

Build the docker image of this new test code, and try to run it as a test pod in the cluster, just like how we run the existing longhorn tests.

And then we know if it has some impact on the existing workflow.

Thanks!

Reply:
I've tried to build negative testing as Pod and run it in the cluster. 
![image](https://github.com/longhorn/longhorn/assets/17548901/1351f2fd-23d9-440b-aeff-e84103bd6abe)

I think there are two tasks need to do, and I have created a ticket https://github.com/longhorn/longhorn/issues/6502 to follow to improve.
 1. Having Dockerfile to build the Docker image
 2. Add ClusterRole: `longhorn.io` in deployment manifest

Reply:
Can you also help to fix the `urllib3 or chardet doesn't match s supported` warning? Thanks!

Reply:
I update the `requests` version in `requirements.txt` as `2.31.0`, the `urllib3 or chardet doesn't match s supported` will be fixed.

![image](https://github.com/longhorn/longhorn-tests/assets/17548901/34afd1b5-5dcc-48a7-930d-13fe462b036d)